pkg/validating-webhooks/hiveextension/v1beta1/agentclusterinstall_admission_hook_test.go

```diff
@@ -25,6 +25,9 @@ func createDecoder() *admission.Decoder {
 	return decoder
 }
 
+var TRUE_VALUE bool = true
```

Comment:
It's not a go style https://go.dev/doc/effective_go#constants 
And i'm not sure that you really need it if you have `swag.Bool`